collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/ServerSettingsTest.java

```diff
@@ -90,4 +75,14 @@ public void selectingGoogleAccount_showsGoogleAccountSettings() {
                 .assertText(R.string.selected_google_account_text)
                 .assertText(R.string.google_sheets_url);
     }
+
+    @Test
+    public void selectingGoogleAccount_disablesAutomaticUpdates() {
+        MainMenuPage mainMenu = new MainMenuPage(rule).assertOnPage()
+                .enablePreviouslyDownloadedOnlyUpdates();
+        assertThat(testDependencies.scheduler.getDeferredTasks().size(), is(1));
+
+        mainMenu.setGoogleDriveAccount("steph@curry.basket");
```

Comment:
How very American.

Reply:
Heh. I'm enjoying the NBA restart but missing my team.